dev-libs/libp11/libp11-0.4.3.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A library that implement a layer on top of PKCS#11 API"
```

Comment:
"that implement"  -> "that implement**s**" or maybe "implementing"

BTW, I think we can improve DESCRIPTION too, but don't worry about this.
If anyone has any suggestion, please do not hesitate!